test/e2e/specs/specs-calypso/wp-gutenboarding-craete-spec.js

```diff
@@ -139,14 +139,4 @@ describe( 'Gutenboarding: (' + screenSize + ')', function () {
 			await new DeleteSiteFlow( driver ).deleteSite( newSiteDomain );
 		} );
 	} );
-
-	describe( 'Visit Gutenboarding page as a logged in user @parallel', function () {
-		it( 'Can log in as user', async function () {
-			await new LoginFlow( driver ).login();
-		} );
-
-		it( 'Can visit Gutenboarding', async function () {
-			await NewPage.Visit( driver );
-		} );
-	} );
 } );
```

Comment:
There is a typo in the file name (`craete`)

Reply:
🤦🏼 
Very good catch, fixed!